docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+name: a12n-server-dev
+
+services:
+  db:
+      image: mysql:8.0
+      restart: always
+      ports:
+        - "3306:3306"
+      environment:
+        - MYSQL_DATABASE=a12nserver
+        - MYSQL_USER=a12nserver
+        - MYSQL_ROOT_PASSWORD=root # NOTE: not recommended for production
+        - MYSQL_PASSWORD='password'
+        - DB_HOST=db
+        - DB_PASSWORD='password'
+        - DB_USER=a12nserver
+        - DB_DATABASE=a12nserver
+  a12n-server:
+    build: 
+      context: .
+      dockerfile: Dockerfile.dev
+    labels:
+      - org.opencontainers.image.source=https://github.com/curveball/a12n-server
+    ports:
+      - "8531:8531"
+    environment:
+      - NODE_ENV=development
+      - PORT=8531
+      - DATABASE_URL=mysql://a12nserver:password@db:3306/a12nserver
+      - JWT_SECRET=secret
+      - CURVEBALL_ORIGIN=http://localhost:8531
```

Comment:
I think this might be optional. It _should_ guess the origin if it's not set!

Reply:
yeah I might not need this